bash_completion.sh

```diff
@@ -128,11 +128,34 @@ _buildtest ()
 {
   local cur="${COMP_WORDS[COMP_CWORD]}"
   local prev="${COMP_WORDS[COMP_CWORD-1]}"
+  declare -i offset="0"
 
   COMPREPLY=()   # Array variable storing the possible completions.
 
-  next=${COMP_WORDS[1]}
+  declare -a buildtest_opts=("--color" "--config" "--debug" "--editor" "--help" "--helpcolor" "--logpath" "--loglevel" "--print-log" "--no-color" "--report" "--version" "--view-log" "-c" "-d" "-h" "-l" "-p" "-r" "-V")
 
+  commands=( "--color" "--config" "-c" "--report" "-r" "--loglevel" "-l" "--editor" )   # Array variable storing commands which require an input argument from the user.
```

Comment:
the bash completion doesnt work with `--color` 

![Screen Shot 2023-04-14 at 11 56 58 AM](https://user-images.githubusercontent.com/12942230/232094946-7b8d8df1-5051-4eaa-b19c-3c3c295ec271.png)

Reply:
can you please test this out with several commands 

- buildtest --color blue bd 
- buildtest --color blue --report python.json bd 
- buildtest --loglevel 
- buildtest --loglevel bd
- buildtest -c /path/to/config bd